src/measure_theory/measure/ae_disjoint.lean

```diff
@@ -71,6 +71,9 @@ measure_mono_null_ae (hu.inter hv) h
 lemma mono (h : ae_disjoint μ s t) (hu : u ⊆ s) (hv : v ⊆ t) : ae_disjoint μ u v :=
 h.mono_ae hu.eventually_le hv.eventually_le
 
+lemma congr (h : ae_disjoint μ s t) (hu : u =ᵐ[μ] s) (hv : v =ᵐ[μ] t) : ae_disjoint μ u v :=
```

Comment:
Should this (and the previous lemma) be protected?

Reply:
Yes, I've added the `protected` attribute in both cases.